__tests__/lint/assertions-enum-error/snapshot.js

```diff
@@ -0,0 +1,26 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`E2E lint assertions-enum-error 1`] = `
+
+validating /openapi.yaml...
+[1] openapi.yaml:20:7 at #/paths/~1pet~1findByStatus/get/summary
+
+Operation summary value should be among of predefined values
```

Comment:
can we show those values in the error message? We also have `suggest` in `report` args, we can use it here maybe.

Reply:
added `suggest`